util/set.go

```diff
@@ -0,0 +1,33 @@
+package util
```

Comment:
`import go.util.collection`?

I don't see why separate this right now, rather wait for the `util` package to grow.